ac/ac-ck-board/src/index.js

```diff
@@ -8,17 +8,12 @@ import { meta, config, configUI } from './meta';
 const dataStructure = [];
 
 const mergeFunction = (object, dataFn) => {
-  [...(object.config.boxes || []), ...(object.data || [])].forEach(box => {
-    if (!box.id) {
-      box.id = uuid();
-    }
-    if (box.title && box.content) {
-      dataFn.listAppend({
-        ...box,
-        x: Math.random() * 800,
-        y: -(Math.random() * 800)
-      });
-    }
+  (object.data || []).forEach(box => {
```

Comment:
Can still configure list of starting boxes, but they won't be used.

We could discuss if starting data of an activity should always come from an operator or it could be part of the configuration somehow.